test/flags/sources.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flags
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Sources holds the Sourcess we want to run test against.
+type Sources []metav1.TypeMeta
+
+func (sources *Sources) String() string {
+	return fmt.Sprint(*sources)
+}
+
+// Set converts the input string to Sources.
```

Comment:
"Appends a new Source corresponding to the input string" seems to describe the actual code better.

Reply:
I'll refactor this in a separate PR and actually rename the method to `Append`. Good catch :+1: . I'm not fully familiar with the code base so I'm not sure who's using who and I wanna limit the blast radius of this PR.

Reply:
Can we at least have an accurate comment in _this_ PR?